audio/filter/af_rubberband.c

```diff
@@ -99,13 +99,23 @@ static int control(struct af_instance *af, int cmd, void *arg)
         return AF_OK;
     case AF_CONTROL_COMMAND: {
         char **args = arg;
+        char *endptr;
+        double pitch, mult;
         if (!strcmp(args[0], "set-pitch")) {
-            char *endptr;
-            double pitch = strtod(args[1], &endptr);
+            pitch = strtod(args[1], &endptr);
             if (*endptr || pitch < 0.01 || pitch > 100.0)
                 return CONTROL_ERROR;
             update_pitch(af, pitch);
             return CONTROL_OK;
+        } else if (!strcmp(args[0], "multiply-pitch")) {
+            mult = strtod(args[1], &endptr);
+            if (*endptr || mult <= 0)
+                return CONTROL_ERROR;
+            pitch = p->pitch * mult;
+            if (pitch < 0.01 || pitch > 100.0)
```

Comment:
why not deduplicate this code by pulling it out of the if?

Reply:
Future af-commands might have nothing to do with pitch, so I put this test in both current af-commands. However I could move it into `static void update_pitch()` instead, and make it a `static bool update_pitch()` instead.

Reply:
Yeah, that might be an idea. It looks like all uses of `update_pitch` are clamped before hand, so this wouldn't change behaviour.

Reply:
sure, looks like there are no uses of update_pitch which don't clamp, so this seems reasonable

Reply:
done